front/src/common/Map/Layers/extensions/SNCF/PSL.tsx

```diff
@@ -10,7 +10,6 @@ import { FilterSpecification } from 'maplibre-gl';
 
 import { MAP_URL } from 'common/Map/const';
 import OrderedLayer from 'common/Map/Layers/OrderedLayer';
-import { getInfraID } from 'reducers/osrdconf/selectors';
 import { MapState } from 'reducers/map';
 import { getSpeedSectionsTag, getSpeedSectionsName } from '../../SpeedLimits';
 import { Theme, OmitLayer } from '../../../../../types';
```

Comment:
Could you add `type` to all other type imports of the file please :)
```suggestion
import type { Theme, OmitLayer } from 'types';
```

Reply:
Done